src/internal_events/loki.rs

```diff
@@ -36,17 +36,16 @@ pub struct LokiOutOfOrderEventDropped;
 
 impl InternalEvent for LokiOutOfOrderEventDropped {
     fn emit_logs(&self) {
-        warn!(
+        debug!(
             message = "Received out-of-order event; dropping event.",
             internal_log_rate_secs = 30
         );
     }
 
     fn emit_metrics(&self) {
         counter!("events_discarded_total", 1,
-                "reason" => "out_of_order");
-        counter!("processing_errors_total", 1,
-                "error_type" => "out_of_order");
+                "reason" => "out_of_order"); // deprecated
+        counter!("component_discarded_events_total", 1);
```

Comment:
I didn't include the `reason` here as that's the only reason that we drop events in this sink (AFAIK)

Reply:
I think it's still a good idea to include it to make it obvious this is not due to some other error.